cli/silence_import.go

```diff
@@ -100,28 +115,16 @@ func bulkImport(cmd *cobra.Command, args []string) error {
 			s.ID = ""
 		}
 
-		silences <- &s
+		silencec <- &s
 		count++
 	}
-	close(silences)
 
-	// read closing bracket
-	_, err = dec.Token()
-	if err != nil {
-		return errors.Wrap(err, "invalid JSON")
```

Comment:
Removed this as I think it doesn't really make sense to throw an error over JSON format here, the silences are added already at this point.

Reply:
I would actually prefer to keep it in. Even though all the silences have been added, a user should still be informed if they are attempting to use invalid json.